nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name(name)
+    name.length >= 10 ? name[0..9].capitalize : name.capitalize
```

Comment:
- [ ] This method should raise a NotImplementedError according to the project requirements. Kindly update it.

Reply:
Actioned 🤓  -> Thanks a lot for sporting this.